analyzers/tests/SonarAnalyzer.Test/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Invocation.Enumerable.cs

```diff
@@ -720,6 +721,7 @@ public void Invocation_CollectionMethods_SetCollectionConstraint()
         Verify("clear", ObjectConstraint.NotNull, CollectionConstraint.Empty);
         Verify("add", ObjectConstraint.NotNull, CollectionConstraint.NotEmpty);
         Verify("remove", ObjectConstraint.NotNull);
+        Verify("addExtension", ObjectConstraint.NotNull, CollectionConstraint.NotEmpty);
```

Comment:
this was not adding notempty before that change in the collectionTracker  that I missed adding a UT for.